app/src/main/res/values/strings.xml

```diff
@@ -74,64 +74,64 @@
-    <string name="prefs_deviceprotecteddata">Backup device protected data</string>
-    <string name="prefs_deviceprotecteddata_summary">Backs up and restores data that is placed in data/user_de</string>
+    <string name="prefs_password_match_false">The passwords don\'t match</string>
+    <string name="prefs_excludecache">Exclude the app\'s cache</string>
+    <string name="prefs_excludecache_summary">Exclude the app\'s cache from the data backup</string>
+    <string name="prefs_externaldata">Back up external data</string>
+    <string name="prefs_externaldata_summary">Backs up and restores data placed on the external storage (e.g. /storage/emulated/0/Android/data/)</string>
+    <string name="prefs_deviceprotecteddata">Back up device protected data</string>
+    <string name="prefs_deviceprotecteddata_summary">Backs up and restores data placed in data/user_de</string>
     <string name="prefs_copyselfapk">Copy own APK</string>
     <string name="prefs_copyselfapk_summary">Place a copy of OAndBackupX in the top level of the backup folder when backed up</string>
     <string name="prefs_acquirewakelock">Prevent sleep</string>
-    <string name="prefs_acquirewakelock_summary">Acquire a wakelock while batch operations are running to prevent sleep. will speed up backups and restores considerably but will consume more energy</string>
-    <string name="prefs_enableSpecialBackups">Enable special backups</string>
+    <string name="prefs_acquirewakelock_summary">Acquire a wakelock while batch operations are running to prevent sleep and speed up backups and restoring considerably by using more energy</string>
+    <string name="prefs_enableSpecialBackups">Turn on special backups</string>
     <string name="prefs_quickreboot">Quick reboot</string>
-    <string name="prefs_quickreboot_summary">Kills the system_server process to force a restart of user space</string>
+    <string name="prefs_quickreboot_summary">Kills the system_server process to force a restart of the app</string>
     <string name="prefs_batchdelete">Delete backups</string>
-    <string name="prefs_batchdelete_summary">Delete backups of all apps not currently installed</string>
-    <string name="prefs_logviewer">View the log</string>
+    <string name="prefs_batchdelete_summary">Install the ability to delete all app backups first</string>
+    <string name="prefs_logviewer">View log</string>
     <string name="prefs_help">Help</string>
 
     <string name="sched_title">Scheduling</string>
     <string name="sched_add">Add Schedule</string>
     <string name="sched_notificationMessage">Scheduled backup</string>
     <string name="sched_interval">Interval in days:</string>
-    <string name="sched_hourOfDay">Hour of day:</string>
-    <string name="sched_checkbox">Enable</string>
+    <string name="sched_hourOfDay">Hour of the day:</string>
```

Comment:
https://english.stackexchange.com/questions/306016/why-is-it-time-of-day-but-hour-of-the-day